registration/forms.py

```diff
@@ -247,7 +247,6 @@ def create_user(self, application, commit=True):
         user = User(
             username=self.cleaned_data['username'],
             email=self.cleaned_data['email'],
-            is_active=False,
```

Comment:
@bradenmacdonald @omarkhan Ok for not overloading the definition of is_active; in that case though, how can we ensure (simply) that users have validated both their email addresses before approving their application? I'd like to avoid ending up with validated approved beta-testers with non validated email addresses.